### DISCUSSION TITLE
[Question]: Marking events to be preserved

### DISCUSSION THREAD

**Discussion by hipitihop** [blakeblackshear/frigate#13407] [Ask A Question] [answered]
### What is your question?

Prior 0.14.0 is there a way to mark an event in the reviews ? Previously one could star an event in the list and it would not be deleted.

**Comment by NickM-27** [comment, marked as answer]
You can right click (desktop) or long press (mobile) to select an item then choose export

**Reply by ThomasCr** [reply]
But thats not really the same?
It asks me to export a time range? I just want to save this event?

Its confusing?!

**Reply by NickM-27** [reply]
it is the same, you are saving a period of time so that the recordings are not deleted. On top of that there are many advantages to exports:
- They are namable so you can remember why you exported / wanted to save it
- Exports are searchable making them easier to find
- The recordings are consolidated into a single file so you don't have to have the frigate database to play them back
- many more

**Reply by NickM-27** [reply]
you can also right click on a specific review item to directly export that item, to be clear
